### ISSUE TITLE
Are there pretrained models on Scenenet only?

### ISSUE THREAD

**Issue by zeroAska** [ankurhanda/pytorch-SceneNetRGBD#2]
Hi, 

Is there a model only trained on Scenenet, but not on NYU/SUN dataset? Say if I wanted to use the pretrained weights on Scenenet and fine-tune on my own dataset, which pretrained weight shall I use on that google drive folder? `nyu_models/rgb_scenenet_pretrain/converted_clean_model.pth` or `sun_models/rgb_scenenet_pretrain/converted_clean_model.pth`?

Thanks!

**Comment by ankurhanda** [comment]
any should be fine.

**Comment by zeroAska** [comment]
Thanks for replying! Are these two models already fine-tuned on nyu/sun respectively?

**Comment by ankurhanda** [comment]
I think so. I will dig up if I still have the scenenet trained models - they should be somewhere but these should be also usable for you.

**Comment by zeroAska** [comment]
Great, thank you so much for the help! We are trying to use the scenenet's trajectories and the trained weight to build 3D semantic maps on its validation set. As scenenet has so many images, we don't have enough resources to train from scratch -- it will be awesome if we can directly use the trained models.
